dateutil/test/test_tz.py

```diff
@@ -1079,6 +1082,28 @@ def testGettzCacheTzLocal(self):
         assert local1 is not local2
 
 
+@pytest.mark.gettz
+def test_gettz_same_result_for_none_and_empty_string():
+    local_from_none = tz.gettz()
+    local_from_empty_string = tz.gettz("")
+    assert local_from_none is not None
+    assert local_from_empty_string is not None
+    assert local_from_none == local_from_empty_string
+
+
+@pytest.mark.gettz
+@pytest.mark.parametrize("gettz_arg", [None, ""]) # could add colon here
+@given(dt=st.datetimes(max_value=datetime.now()))
+def test_gettz_returns_local(gettz_arg, dt):
+    dt_act = dt.replace(tzinfo=tz.gettz())
```

Comment:
Since `datetime` bounds have to be naive I instead initialized the constant from a timestamp.

The test now also has a minimum bound, since I ran into an issue when `dt.astimezone()` started returning LMT (solar time?) in the 19th century.

This looks like a similar issue to the one with the upper bound, although I assume the problem in the case is probably that my local zone file starts at 1901 (`zdump -v /etc/localtime | head -n 1`), after the transition from LMT was already over, while `astimezone()` gets more historical information?

If that is the case I am not sure which date would be an appropriate lower bound, other than 1970.